tests/Filters/AppliesFiltersTraitTest.php

```diff
@@ -13,6 +13,13 @@
 
 class AppliesFiltersTraitTest extends TestCase
 {
+    public function tearDown(): void
+    {
+        parent::tearDown();
+
+        Mockery::close();
+    }
```

Comment:
Lets use the mockery phpunit trait to close Mockery, it's simpler and requires less lines. Main reason why the trait is better is because the entire closing business is hidden. You simply state that your test integrates with Mockery and it handles all the specifics under the hood.